smart_room/endpoints/aircon/model.py

```diff
@@ -0,0 +1,12 @@
+from sqlalchemy.dialects.postgresql import UUID
+
+from config import db
+from utils.base_models import SmartRoomBaseMixin
+
+
+class Aircon(db.Model, SmartRoomBaseMixin):
+    __tablename__ = "aircon"
+    action_front = db.Column(db.String(80))
+    action_back = db.Column(db.String(80))
```

Comment:
Maybe we need a comment to explain the meaning of front and end.

Reply:
Fix in 41db75e.